### PR TITLE
DCOS-45801: handle cosmos errors

### PR DESCRIPTION
**Pull request by nLight**
Cherry-picking #3431 

Just checking @natmegs I _should_ recompile the translations as I'm touching the code, right? Basically I should be recompiling them always?

### CODE REVIEW

**Comment by natmegs** [comment]
Yes for any PR `npm run lingui-extract-with-plugins` needs to be run and any changes to `messages.json` (en and zh) should be committed. Even if text content is not changing line numbers can :)

File: locale/en/messages.json [hunk 1]
```diff
@@ -109,42 +109,6 @@
       ]
     ]
   },
-  "1 Backend": {
```
Comment (DanielMSchmidt) [inline locale/en/messages.json]: That's a lot of deletion..

Comment (DanielMSchmidt) [reply locale/en/messages.json]: We should find a better solution for storing these, it seems to add quite a lot of accidental complexity

Comment (nLight) [reply locale/en/messages.json]: I run `lingui-extract-with-plugins` and this is what I get with no errors... @natmegs any idea? 🙏

Comment (nLight) [reply locale/en/messages.json]: Oh, gotcha... some adjustments to the library should be made. Very helpful wiki page!

**Pull request merged**